repos/bluerobotics/cockpit/metadata.json

```
@@ -0,0 +1,6 @@
+{
+    "name": "Cockpit",
+    "website": "https://github.com/bluerobotics/cockpit",
+    "docker": "patrickelectric/cockpit:latest",
```

The tags are fetched automatically by CI. 

```suggestion
    "docker": "patrickelectric/cockpit",
```

---

Done.